EngineBay.Auditing.Tests/Bases/BaseTestWithDbContext.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace EngineBay.Auditing.Tests
+{
+    using EngineBay.Persistence;
+    using Microsoft.EntityFrameworkCore;
+
+    public class BaseTestWithDbContext<TContext> : IDisposable
+        where TContext : ModuleDbContext
+    {
+        private bool isDisposed;
+
+        protected BaseTestWithDbContext(string databaseName)
+        {
+            var dbContextOptions = new DbContextOptionsBuilder<ModuleWriteDbContext>()
+                    .UseInMemoryDatabase(databaseName)
+                    .EnableSensitiveDataLogging()
+                    .Options;
+
+            var context = Activator.CreateInstance(typeof(TContext), dbContextOptions) as TContext;
+            ArgumentNullException.ThrowIfNull(context);
```

Comment:
For consistency rather use
`if (context is null)
{
throw new ArgumentException(nameof(context));
}`